main.tex

```diff
@@ -219,10 +219,12 @@ \subsection{Astropy development model}
 commits.
 \figurename~\ref{fig:ncommits}, left, shows the distribution of total number of
 commits per contributor as of November 2017.
-This distribution shows that the \astropypkg core package has been developed by
-a broad contributor base, with a core team of developers having added over 1000
-commits to the repository, and a smooth distribution of contribution level down
-to a single commit.
+The relative flatness of this distribution (as demonstrated by its log-slope of 
+-0.5) shows that the \astropypkg core package has been developed by a broad
+contributor base.  A leading group of 6 developers have each added over 1000
+commits to the repository, with $\sim 20$ more in a core team with at least 
+100.  But there is a smooth distribution of contribution level from 100 commits
```

Comment:
@eteq - ah, I was actually referring to the first 100 in the line. Could you commits there too?

Reply:
I'll merge and fix on my branch